specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-02-01/management.JSON

```diff
@@ -0,0 +1,2000 @@
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetGroupSettings": {
+            "$ref": "./examples/GetHierarchySettings.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ManagementGroups"
+        ],
+        "operationId": "HierarchySettings_Put",
+        "description": "Creates or updates the hierarchy settings defined at the Management Group level.\n",
+        "parameters": [
+          {
+            "$ref": "#/parameters/GroupIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/CreateOrUpdateManagementGroupSettingsRequestParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/HierarchySettings"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetGroupSettings": {
+            "$ref": "./examples/PutHierarchySettings.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "ManagementGroups"
+        ],
+        "operationId": "HierarchySettings_Patch",
+        "description": "Updates the hierarchy settings defined at the Management Group level.\n",
+        "parameters": [
+          {
+            "$ref": "#/parameters/GroupIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/CreateOrUpdateManagementGroupSettingsRequestParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/HierarchySettings"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetGroupSettings": {
+            "$ref": "./examples/PatchHierarchySettings.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ManagementGroups"
+        ],
+        "operationId": "HierarchySettings_Delete",
+        "description": "Deletes the hierarchy settings defined at the Management Group level.\n",
+        "parameters": [
+          {
+            "$ref": "#/parameters/GroupIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
204 should be added

Reply:
Same as above, we have changed our status code response to respect RPC. Why should 204 be added back since we are not returning it anymore with this API version?

Reply:
[RPC](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#delete-resource) says the following. 204 must be added for deletes.
and a 204 (NoContent) should be used if the resource does not exist and the request is well formed.

---
In reply to: [388539415](https://github.com/Azure/azure-rest-api-specs/pull/8443#discussion_r388539415) [](ancestors = 388539415)

Reply:
204 is still missing

---
In reply to: [389115803](https://github.com/Azure/azure-rest-api-specs/pull/8443#discussion_r389115803) [](ancestors = 389115803,388539415)